Description: accept newer minor version of crate addr2line
Author: Greg Lamberson <greg@lamco.io>
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/1140668
Last-Update: 2026-07-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -216,7 +216,7 @@
 wit-smith = { workspace = true, features = ["clap"], optional = true }
 
 # Dependencies of `addr2line`
-addr2line = { version = "0.24.0", optional = true }
+addr2line = { version = ">= 0.24.0, <= 0.25", optional = true }
 gimli = { workspace = true, optional = true }
 
 # Dependencies of `wast`
